### PR TITLE
feat(LLD) Post Onboarding New Items

### PR DESCRIPTION
**Pull request by mle-gall**
### 📝 Description

New post onboarding items for LLD (same as LLM, see[ Live-9559](https://github.com/LedgerHQ/ledger-live/pull/5458))

https://github.com/LedgerHQ/ledger-live/assets/39890664/b793b29a-34dc-4eb3-9e43-a4aa0bf6f16d

### ❓ Context

- **JIRA or GitHub link**: [LIVE-9823]
### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-9823]: https://ledgerhq.atlassian.net/browse/LIVE-9823?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by KVNLS** [comment]
I have a Stax without any app and a LL without BTC account. When I want to buy some crypto, the add account never display the newly added account. + We have a display issue with the back button:

https://github.com/LedgerHQ/ledger-live/assets/31533861/b427c2db-6a1f-447a-b309-607be15b20dd

**Comment by KVNLS** [comment]
I have a Stax with a BTC app and a BTC account added to LL. When I want to buy some crypto and I select my BTC account, the drawer autoclose

https://github.com/LedgerHQ/ledger-live/assets/31533861/39fdb3b1-03d0-4590-a8ac-e7b38100646e

 after the selection and I can't continue the flow

**Comment by KVNLS** [comment]
Same use case as above. When I close the drawer and I reopen it the state is not initialized + I can't go back and I can't close the drawer.

https://github.com/LedgerHQ/ledger-live/assets/31533861/5a0d0349-73e6-4e22-8ff4-f6479b293650

**Comment by mle-gall** [comment]
Hello @KVNLS
After research and debugging last Friday, I found out that opening the live-app within a side drawer is not as straightforward as it seemed (there are context infos passed by the view it's within to make sure the live-app works properly).
Opening the live-app in a drawer wasn't actually required from the ticket which expected a redirect to the buy flow, so I opted to navigate from the post onboarding to the buy flow when clicked.
It's less ideal as it closes the post-onboarding but I think this trade off still makes sure that the flows are fully usable.

File: apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx [hunk 1]
```diff
@@ -1,61 +1,99 @@
...
+        currency: null,
+      }),
     );
-  }
-}
+  }, [dispatch]);
+
+  useEffect(() => {
+    if (!hasAccounts) {
+      console.log("openingAddAccount");
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx:70]: 🗑️

File: apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx [hunk 2]
```diff
@@ -1,61 +1,99 @@
...
-  handleStepChange = (stepId: StepId) =>
-    this.setState({
-      stepId,
-    });
-
-  handleChangeAddressVerified = (isAddressVerified?: boolean | null, err?: Error | null) => {
+const ReceiveModal = () => {
+  const [state, setState] = useState<State>(INITIAL_STATE);
+
+  // You can access state properties like this:
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx:23]: ChatGPT prompt? :p

File: apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx (hunk 1 above)
Comment (mcayuelas-ledger) [inline apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx:70]: 🗑️

**Review by KVNLS — APPROVED** [review]
Works well! Just one console.log to remove and we're good, thanks!

File: apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx (hunk 2 above)
Comment (mle-gall) [reply apps/ledger-live-desktop/src/renderer/modals/Receive/index.tsx:23]: To convert from Class comp to functionnal comp yes ! Cleaning up the comments tho

**Pull request merged**